samples/snippets/csharp/VS_Snippets_CLR/conceptual.disposable/cs/dispose1.cs

```diff
@@ -12,10 +12,8 @@ public class Disposable : IDisposable
    // <Snippet7>
    public void Dispose()
    {
-      // Dispose of unmanaged resources.
-      Dispose(true);
-      // Suppress finalization.
-      GC.SuppressFinalize(this);
+      Dispose(true);             // Dispose of unmanaged resources.
+      GC.SuppressFinalize(this); // Suppress finalization.
```

Comment:
Is there a style guideline for samples that say to use aligned, trailing comments?  If not, I'd revert this.  Preceding comments aligned as if they were code is by far the norm.